src/components/Table/types.ts

```diff
@@ -75,15 +75,20 @@ export type GridColumn<R extends Kinded> = {
   canHide?: boolean;
   /** Flag that will allow to know which hide-able columns are visible on initial load */
   initVisible?: boolean;
-  /** A list of columns that should only be shown when this column is "expanded" */
-  expandColumns?: GridColumn<R>[];
+  /**
+   * Either
+   * 1. A list of columns that should only be shown when this column is "expanded", or...
+   * 2. `number | string` representing width to allow for the column to expand itself larger. This prop is treated the same as the `GridColumn.w` prop.
+   *     Example: Collapsed state shows number of books. Expanded state shows titles of books.
+   */
```

Comment:
Thought: Huh...seems like maybe this should just be a different key? Like `expandedSize`? 

Even then, the name `expandedSize` would make me think its related to "expanded-as-in-adding-new columns", but AFAICT there are basically totally separate features? They just both happen to use the term "expanded"?

Should we use a totally separate term for this new feature...

Reply:
Yea, I was wondering about that too. My hesitation was the simplicity in using the same prop to signal the same/similar behavior. I also thought about renaming `expandColumns` to something else that better noted it could be either more columns of a size value.

Reply:
Is the intent to use this same "expanded-becomes-wider" and "expanded-shows-more-columns" in the same table? Just curious...

Do they want the "expanded-becomes-wider" to be useable in tables that don't even use the "expanded-show-more-columns" feature? Like in a regular table with just a regular `header`, similar to sorting, I can like toggle it expanded-becomes-wider?

Reply:
Yes, that is the intent. We could have a column that shows a count of "Add-on". When expanded, it'll list out the Add-on names.
The UX is the same between the two, they look no different. The result is whether expanded increases the width of a single column, or adds more columns. Though, if we did use a `expandedSize` prop, then we could do both... allow for "sub-columns" to show when expanded, and define a different width for the column that became expanded, which would be really nice. 
I'll go ahead and update to use a `expandedWidth` for this purpose.

Reply:
Updated to use a different property. Thanks for the feedback.